debugview/plugin.cpp

```diff
@@ -179,8 +179,7 @@ class debugview : public threadloop {
 		}
 		if(last_datum_with_images->img0.has_value()){
 			glBindTexture(GL_TEXTURE_2D, camera_textures[0]);
-			cv::Mat img0;
-			cv::cvtColor(*last_datum_with_images->img0.value(), img0, cv::COLOR_BGR2GRAY);
```

Comment:
Please make a corresponding change in `offline_imu_cam` (insert the `cvtColor` call or read the images in grayscale in the first place).